.github/workflows/deploy-adventist-cluster.yaml

```diff
@@ -1,5 +1,5 @@
 name: "Deploy adventist cluster"
-run-name: Deploy (${{ github.ref_name }} -> ${{ inputs.environment }}) by @${{ github.actor }}
+run-name: Deploy ${{ inputs.environment }} ${{ github.ref_name }}@${{ github.sha }} by @${{ github.actor }}
```

Comment:
I am hesitant to add this across all projects until we have fully tested out. I personally would like to find a solution to the length of the sha to be 8 digits, but that is also me being nit-picky and am open to conversations about pros/cons.